dss/stepfunctions/__init__.py

```diff
@@ -5,6 +5,43 @@
 import boto3
 
 
+def step_functions_arn(state_machine_name_template: str) -> str:
+    """
+    The ARN of a state machine, with name derived from `state_machine_name_template`, with string formatting to
+    replace {stage} with the dss deployment stage.
+    :param state_machine_name_template:
+    :return:
+    """
+
+    sts_client = boto3.client("sts")
+
+    region = boto3.Session().region_name
+    accountid = sts_client.get_caller_identity()['Account']
```

Comment:
Here and elsewhere, please use `dss.util.aws.ARN.get_region()` and `dss.util.aws.ARN.get_account_id()`. This avoids an expensive network call every time you run this function. In general, please make sure you watch for unnecessary network calls.